cluster-state-service/swagger/v1/swagger.json

```diff
@@ -248,8 +264,30 @@
     }
   },
   "definitions": {
+    "Metadata": {
+      "type": "object",
+      "required": [
+        "entityVersion"
+      ],
+      "properties": {
+        "entityVersion": {
+          "type": "string"
+        }
+      }
+    },
     "ContainerInstance": {
       "type": "object",
+      "properties": {
+        "metadata": {
+          "$ref": "#/definitions/Metadata"
```

Comment:
I've gone back and forth on this. Right now, the metadata is the same for both objects, as it's just entityVersion. If you have a strong opinion for separating them out, i'll make different objects for each.